### PR TITLE
Add Material Transport Env

### PR DESCRIPTION
**Pull request by shalin-jain**
Sample DAgger run: https://wandb.ai/star-lab-gt/JaxMARL/runs/lj5zszf4?nw=nwusershalin

### CODE REVIEW

File: baselines/QLearning/config/config.yaml [hunk 1]
```diff
@@ -13,4 +13,4 @@ SAVE_PATH: baselines/QLearning/checkpoints
 VISUALIZE_FINAL_POLICY: True
 # number of seeds/envs to visualize
 VIZ_NUM_ENVS: 3
-VIZ_NUM_SEEDS: 5
+VIZ_NUM_SEEDS: 5
```
Comment (kfu02) [inline baselines/QLearning/config/config.yaml]:
```suggestion
```

Comment (kfu02) [reply baselines/QLearning/config/config.yaml]: (an update to `qmix.py` visualizes over all NUM_SEEDS now)

File: baselines/QLearning/config/env/mpe_simple_transport.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  dropoff_reward: 10 # the amount of reward an agent gets for dropping off a resource
+  quota_penalty: -0.1
+
+  # capabilities implemented in MPE_simple_fire: [fire_fight_cap, accel]
+  capability_aware: True # T/F
+  num_capabilities: 2
+  num_agents: 4
+
+  # NOTE: if these lists are longer than num_agents, env will sample num_agents randomly from them each train iter
+  # NOTE: hijacking agent_rads = firefighting capability
```
Comment (kfu02) [inline baselines/QLearning/config/env/mpe_simple_transport.yaml]:
```suggestion
```

File: baselines/QLearning/config/env/mpe_simple_transport.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ENV_KWARGS:
+  max_steps: 100 # 25
+
+  # reward shaping
+  lumber_pickup_reward: 2.5 # reward given to an agent for pickup up lumber if their lumber capacity is > 0
+  concrete_pickup_reward: 2.5 # reward given to an agent for pickup of concrete if their capacity is > 0.
+  dropoff_reward: 10 # the amount of reward an agent gets for dropping off a resource
+  quota_penalty: -0.1
+
+  # capabilities implemented in MPE_simple_fire: [fire_fight_cap, accel]
```
Comment (kfu02) [inline baselines/QLearning/config/env/mpe_simple_transport.yaml]:
```suggestion
  # capabilities implemented in MPE_simple_transport: [lumber_cap, concrete_cap]
```

**Review by kfu02 — APPROVED** [review]
This is great, Shalin! Before merging could you (1) do a 1-seed run of DAgger with the normal RNN (not hyperaware) and (2) do a quick regression test on the fire env for both QMIX and DAgger (just make sure both still run the fire env correctly, don't have to train fully to completion). Thanks!

**Comment by shalin-jain** [comment]
> This is great, Shalin! Before merging could you (1) do a 1-seed run of DAgger with the normal RNN (not hyperaware) and (2) do a quick regression test on the fire env for both QMIX and DAgger (just make sure both still run the fire env correctly, don't have to train fully to completion). Thanks!

Added these runs under the tag `test-regression`!

**Pull request merged**